apps/test/unit/code-studio/pd/professional_learning_landing/EnrolledWorkshopsTest.jsx

```diff
@@ -92,15 +117,15 @@ describe('EnrolledWorkshops', () => {
     );
   });
 
-  it('Clicking "Print Certificate" opens the certificate in a new tab', function() {
+  it('Clicking "Print Certificate" opens the certificate in a new tab if user attended workshop', function() {
     const enrolledWorkshopsTable = shallow(
       <EnrolledWorkshopsTable workshops={workshops} />
     );
 
     // Click the "Print Certificate" button
     enrolledWorkshopsTable
-      .find('tr')
-      .last()
+      .find('tbody tr')
```

Comment:
Changed this `find` call because there is a header row that had me banging my head against the wall for a bit.